### PR TITLE
app-admin/syslog-ng: depend on dev-libs/libpcre2 instead of dev-libs/libpcre

### PR DESCRIPTION
**Pull request by cfuga**
app-admin/syslog-ng changed this dependency since version 4.3.0

https://github.com/syslog-ng/syslog-ng/commit/cb6de08dc9078644d48ca536b5660e406b1a50d6

Signed-off-by: Cristian Othón Martínez Vera <cfuga@cfuga.mx>

### CODE REVIEW

**Comment by Flowdalic** [comment]
Needs revision bump

File: app-admin/syslog-ng/syslog-ng-4.3.1.ebuild [hunk 1]
```diff
@@ -21,7 +21,7 @@ RESTRICT="!test? ( test )"
 RDEPEND="
 	>=dev-libs/glib-2.10.1:2
 	>=dev-libs/ivykis-0.42.4
-	>=dev-libs/libpcre-6.1
+	>=dev-libs/libpcre2-10.0
```
Comment (Flowdalic) [inline app-admin/syslog-ng/syslog-ng-4.3.1.ebuild:24]:
```suggestion
	>=dev-libs/libpcre2-10.0:=
```

File: app-admin/syslog-ng/syslog-ng-4.4.0.ebuild [hunk 2]
```diff
@@ -21,7 +21,7 @@ RESTRICT="!test? ( test )"
 RDEPEND="
 	>=dev-libs/glib-2.10.1:2
 	>=dev-libs/ivykis-0.42.4
-	>=dev-libs/libpcre-6.1
+	>=dev-libs/libpcre2-10.0
```
Comment (Flowdalic) [inline app-admin/syslog-ng/syslog-ng-4.4.0.ebuild:24]:
```suggestion
	>=dev-libs/libpcre2-10.0:=
```

**Review by Flowdalic — CHANGES_REQUESTED** [review]
Thanks for your contribution. This also needs a revision bump.

File: app-admin/syslog-ng/syslog-ng-4.3.1.ebuild (hunk 1 above)
Comment (thesamesam) [reply app-admin/syslog-ng/syslog-ng-4.3.1.ebuild:24]: Fwiw the subslot on linpcre2 is only for one of its installed libraries so need to check if using that

**Comment by cfuga** [comment]
> Thanks for your contribution. This also needs a revision bump.

I've updated the patch with the required revision bump.

File: app-admin/syslog-ng/syslog-ng-4.3.1.ebuild (hunk 1 above)
Comment (thesamesam) [reply app-admin/syslog-ng/syslog-ng-4.3.1.ebuild]: If it only links with pcre2-8, then please don't.

Comment (cfuga) [reply app-admin/syslog-ng/syslog-ng-4.3.1.ebuild:24]: > If it only links with pcre2-8, then please don't.

Ok. I updated the patch and removed the subscription to the subslot.

**Comment by thesamesam** [comment]
Thank you!

File: app-admin/syslog-ng/syslog-ng-4.3.1.ebuild (hunk 1 above)
Comment (Flowdalic) [reply app-admin/syslog-ng/syslog-ng-4.3.1.ebuild:24]: > Fwiw the subslot on linpcre2 is only for one of its installed libraries so need to check if using that

TIL. Thanks for pointing this out. And it's even documented in libpcre2's metadata.xml. \o/

**Pull request closed without merging**